src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/__tests__/HomeHealthCertifiedPersonCellModelTests.swift

```diff
@@ -6,7 +6,7 @@ import XCTest
 import HealthCertificateToolkit
 @testable import ENA
 
-class HomeHealthCertifiedPersonCellModelTests: XCTestCase {
+class HealthCertifiedPersonCellModelTests: XCTestCase {
 
 	func testGIVEN_completelyProtectedCertifiedPerson_THEN_IsSetupCorrectly() throws {
```

Comment:
I think we only need to keep one of the tests as the vaccination state has no influence on the cell anymore

Reply:
yeah, I removed the other two